tests/travis/prepare_functional_parallel.sh

```diff
@@ -6,20 +6,19 @@
 set -e
 trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
 
-test_set_list=$(grep "php72" -Rl src/Test/Functional/Acceptance | sort)
+test_set_list=$(grep -Rc 'php70\|php71' src/Test/Functional/Acceptance | grep ':0$' | sed 's/..$//' | sort)
 test_set_count=$(printf "$test_set_list" | wc -l)
 let "test_set_count++"
 
-test_set_size[1]=$(printf "%.0f" $(echo "$test_set_count*0.33" | bc))
-test_set_size[2]=$(printf "%.0f" $(echo "$test_set_count*0.33" | bc))
+test_set_size[1]=$(printf "%.0f" $(echo "$test_set_count*0.20" | bc))
+test_set_size[2]=$(printf "%.0f" $(echo "$test_set_count*0.40" | bc))
```

Comment:
Why did you make groups unequal?

Reply:
I did this because different tests take a different time because they have a different number of variations.
In the future, I want to split tests by variations.